### ISSUE TITLE
Unable to load a model to continue training

### ISSUE THREAD

**Issue by roboserg** [Unity-Technologies/ml-agents#2032]
So I do `mlagents-learn config/config.yaml --load --run-id=Lunar2D-5 --train`, obvisouly I have the folder named "Lunar2D-5" with the model inside it - https://puu.sh/DruAS/7ed097c264.png

I get an error - https://puu.sh/DruxN/4ff4e2f2aa.png + https://puu.sh/Druy8/d176d28fa8.png

`INFO:mlagents.trainers:{'--base-port': '5005',
 '--curriculum': 'None',
 '--debug': False,
 '--docker-target-name': 'None',
 '--env': 'None',
 '--help': False,
 '--keep-checkpoints': '5',
 '--lesson': '0',
 '--load': True,
 '--no-graphics': False,
 '--num-envs': '1',
 '--num-runs': '1',
 '--run-id': 'Lunar2D-5',
 '--save-freq': '50000',
 '--seed': '-1',
 '--slow': False,
 '--train': True,
 '<trainer-config-path>': 'config/config.yaml'}
c:\users\roboserg\anaconda3\envs\ml-agents\lib\site-packages\mlagents\trainers\learn.py:141: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  trainer_config = yaml.load(data_file)
INFO:mlagents.envs:Start training by pressing the Play button in the Unity Editor.
INFO:mlagents.envs:
'Academy' started successfully!
Unity Academy name: Academy
        Number of Brains: 2
        Number of Training Brains : 1
        Reset Parameters :

Unity brain name: PlayerBrain
        Number of Visual Observations (per agent): 0
        Vector Observation space size (per agent): 5
        Number of stacked Vector Observation: 1
        Vector Action space type: continuous
        Vector Action space size (per agent): [3]
        Vector Action descriptions: Thrust, Left, Right
Unity brain name: LunarBrain2D
        Number of Visual Observations (per agent): 0
        Vector Observation space size (per agent): 5
        Number of stacked Vector Observation: 1
        Vector Action space type: continuous
        Vector Action space size (per agent): [3]
        Vector Action descriptions: , ,
2019-05-13 00:16:47.540108: I T:\src\github\tensorflow\tensorflow\core\platform\cpu_feature_guard.cc:140] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2
2019-05-13 00:16:47.735349: I T:\src\github\tensorflow\tensorflow\core\common_runtime\gpu\gpu_device.cc:1344] Found device 0 with properties:
name: GeForce GTX 1070 major: 6 minor: 1 memoryClockRate(GHz): 1.797
pciBusID: 0000:01:00.0
totalMemory: 8.00GiB freeMemory: 6.64GiB
2019-05-13 00:16:47.741990: I T:\src\github\tensorflow\tensorflow\core\common_runtime\gpu\gpu_device.cc:1423] Adding visible gpu devices: 0
2019-05-13 00:16:48.544922: I T:\src\github\tensorflow\tensorflow\core\common_runtime\gpu\gpu_device.cc:911] Device interconnect StreamExecutor with strength 1 edge matrix:
2019-05-13 00:16:48.549301: I T:\src\github\tensorflow\tensorflow\core\common_runtime\gpu\gpu_device.cc:917]      0
2019-05-13 00:16:48.552560: I T:\src\github\tensorflow\tensorflow\core\common_runtime\gpu\gpu_device.cc:930] 0:   N
2019-05-13 00:16:48.557318: I T:\src\github\tensorflow\tensorflow\core\common_runtime\gpu\gpu_device.cc:1041] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 6416 MB memory) -> physical GPU (device: 0, name: GeForce GTX 1070, pci bus id: 0000:01:00.0, compute capability: 6.1)
INFO:mlagents.trainers:Loading Model for brain LunarBrain2D
INFO:tensorflow:Restoring parameters from ./models/Lunar2D-5-0/LunarBrain2D\model-100004.cptk
INFO:mlagents.envs:Hyperparameters for the PPO Trainer of brain LunarBrain2D:
        batch_size:     10
        beta:   0.005
        buffer_size:    100
        epsilon:        0.2
        gamma:  0.99
        hidden_units:   128
        lambd:  0.95
        learning_rate:  0.0003
        max_steps:      100.0e3
        normalize:      True
        num_epoch:      3
        num_layers:     2
        time_horizon:   64
        sequence_length:        64
        summary_freq:   500
        use_recurrent:  False
        summary_path:   ./summaries/Lunar2D-5-0_LunarBrain2D
        memory_size:    256
        use_curiosity:  False
        curiosity_strength:     0.01
        curiosity_enc_size:     128
        model_path:     ./models/Lunar2D-5-0/LunarBrain2D
2019-05-13 00:16:49.216060: I T:\src\github\tensorflow\tensorflow\core\common_runtime\gpu\gpu_device.cc:1423] Adding visible gpu devices: 0
2019-05-13 00:16:49.221346: I T:\src\github\tensorflow\tensorflow\core\common_runtime\gpu\gpu_device.cc:911] Device interconnect StreamExecutor with strength 1 edge matrix:
2019-05-13 00:16:49.227523: I T:\src\github\tensorflow\tensorflow\core\common_runtime\gpu\gpu_device.cc:917]      0
2019-05-13 00:16:49.231106: I T:\src\github\tensorflow\tensorflow\core\common_runtime\gpu\gpu_device.cc:930] 0:   N
2019-05-13 00:16:49.235661: I T:\src\github\tensorflow\tensorflow\core\common_runtime\gpu\gpu_device.cc:1041] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 6416 MB memory) -> physical GPU (device: 0, name: GeForce GTX 1070, pci bus id: 0000:01:00.0, compute capability: 6.1)
INFO:mlagents.envs:Saved Model
INFO:mlagents.trainers:List of nodes to export for brain :LunarBrain2D
INFO:mlagents.trainers: is_continuous_control
INFO:mlagents.trainers: version_number
INFO:mlagents.trainers: memory_size
INFO:mlagents.trainers: action_output_shape
INFO:mlagents.trainers: action
INFO:mlagents.trainers: action_probs
INFO:mlagents.trainers: value_estimate
2019-05-13 00:16:50.483667: I T:\src\github\tensorflow\tensorflow\core\common_runtime\gpu\gpu_device.cc:1423] Adding visible gpu devices: 0
2019-05-13 00:16:50.487435: I T:\src\github\tensorflow\tensorflow\core\common_runtime\gpu\gpu_device.cc:911] Device interconnect StreamExecutor with strength 1 edge matrix:
2019-05-13 00:16:50.491813: I T:\src\github\tensorflow\tensorflow\core\common_runtime\gpu\gpu_device.cc:917]      0
2019-05-13 00:16:50.496065: I T:\src\github\tensorflow\tensorflow\core\common_runtime\gpu\gpu_device.cc:930] 0:   N
2019-05-13 00:16:50.499464: I T:\src\github\tensorflow\tensorflow\core\common_runtime\gpu\gpu_device.cc:1041] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 6416 MB memory) -> physical GPU (device: 0, name: GeForce GTX 1070, pci bus id: 0000:01:00.0, compute capability: 6.1)
INFO:tensorflow:Restoring parameters from ./models/Lunar2D-5-0/LunarBrain2D\model-100005.cptk
INFO:tensorflow:Froze 20 variables.
Converted 20 variables to const ops.
Converting ./models/Lunar2D-5-0/LunarBrain2D/frozen_graph_def.pb to ./models/Lunar2D-5-0/LunarBrain2D.nn
IGNORED: Cast unknown layer
IGNORED: StopGradient unknown layer
GLOBALS: 'is_continuous_control', 'version_number', 'memory_size', 'action_output_shape'
IN: 'vector_observation': [-1, 1, 1, 5] => 'sub_3'
IN: 'epsilon': [-1, 1, 1, 3] => 'mul_1'
OUT: 'action', 'action_probs', 'value_estimate'
DONE: wrote ./models/Lunar2D-5-0/LunarBrain2D.nn file.
INFO:mlagents.trainers:Exported ./models/Lunar2D-5-0/LunarBrain2D.nn file
`

**Comment by roboserg** [comment]
You have to increase the max_steps size on your yaml config.
